lib/style/deprecations.ex

```diff
@@ -58,6 +58,21 @@ defmodule Styler.Style.Deprecations do
     defp style({:|>, m, [lhs, {{:., _, [{_, _, [mod]}, :slice]} = f, funm, [{:.., _, [_, _]} = range]}]})
          when mod in [:Enum, :String],
          do: {:|>, m, [lhs, {f, funm, [add_step_to_decreasing_range(range)]}]}
+
+    # ~R is deprecated in favor of ~r
+    defp style({:sigil_R, m, args}), do: {:sigil_r, m, args}
```

Comment:
i'd thought about the ~R one but understood it as a breaking change to go from R to r due to different semantics, is that not the case?

Reply:
Based on this [commit](https://github.com/elixir-lang/elixir/commit/3ed3bf9108e4c878e878b4ad040f4e5a8d1212e9) is a simple replacement.